docs/store/index.html

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+title: Store
+permalink: /store
+order: 40
+---
+
+<!-- HEADER -->
+{% include header.html with_signup_form='true' has_sticky_navigation='true' background_color='#7F7E9E' %}
+
+<style>{% include css/store.css %}</style>
+
+<div class="hero">
+  <h1 class="heading">Stack and Stencil Hub</h1>
+  <h2 class="subheading">Start building your stacks easily with these hundereds of examples</h2>
+  <div class="search-container">
+    <div class="dropdown">
+      <a href="#" class="dropdown-selection">
+        <label>In Stencils</label><span class="dropdown-icon"></span>
```

Comment:
for decorative elements such as this `<span class="dropdown-icon"></span>` please use pseudo elements like `:before` or `:after`